test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
I still think that no need to implemented long id because that app is just a demo and it will no work with real user, for the second one i am absolutely agree